### ISSUE TITLE
[Request] Support for Flow Meter

### ISSUE THREAD

**Issue by spaelectronics** [reef-pi/reef-pi#1106]
I would like to see support for more sensors, such as a Flow Meter.

**Comment by gminton86** [comment]
Yes and current sensors like the apex would be awesome so we know if something has failed in our tank

**Comment by ranjib** [comment]
Any specific flow meter you have in mind?

**Comment by spaelectronics** [comment]
Something along the lines of this:
https://www.amazon.com/dp/B00VKAT9VA/

**Comment by Cursed-DIY** [comment]
I would also like to be able to use the above flow meter

**Comment by ranjib** [comment]
This will involve writing a HAL driver for flow meters. It does not look like super complicated... but its not on my priority list. We'll see if we can pick it up for 4.0 release, otherwise definitely for 5.0 
https://www.raspberrypi.org/forums/viewtopic.php?t=118230
https://learn.adafruit.com/adafruit-keg-bot?view=all

**Comment by ranjib** [comment]
for reference: https://www.reef2reef.com/threads/help-with-flow-meter-stopped-working.921202/
